assets/helpers/tracking/googleTagManager.js

```diff
@@ -0,0 +1,26 @@
+import uuidv4 from 'uuid';
+import { forCountry } from '../internationalisation/currency';
+import { get as getCookie } from '../cookie';
+
+// Generates a uuid to anonymously identify this user in this session.
+// We need this to prevent page refreshes on conversion pages being
+// treated as new conversions
+export function getOrderId() {
+  let orderId = sessionStorage.getItem('orderId');
+  if (orderId === null) {
+    orderId = uuidv4();
+    sessionStorage.setItem('orderId', orderId);
```

Comment:
This will [blow up](https://stackoverflow.com/questions/21159301/quotaexceedederror-dom-exception-22-an-attempt-was-made-to-add-something-to-st) in private browsing mode in Safari <11. It's a known bug that leaks private browsing mode, which seems to be fixed in [Safari 11](https://bugs.webkit.org/show_bug.cgi?id=157010).

One workaround is to wrap calls to `sessionStorage` in a `try...catch`. We have a `storage` utility that performs a slightly half-baked check for this, I'd like to standardise it so we can just call a function that handles the problem in future.

Reply:
@Ap0c is this the same for sessionStorage do you know?

Reply:
Yup, looks like it is

Reply:
Yeah it's both, hit the same problem in #252.